tests/cli_autogen_input/sonic-1-list.yang

```diff
@@ -0,0 +1,29 @@
+module sonic-1-list {
+
+    yang-version 1.1;
+
+	namespace "http://github.com/Azure/s-1-list";
+	prefix s-1-list;
+
+	container sonic-1-list {
+		/* sonic-1-list - top level container */
+
+		container TABLE_1 {
+			/* TABLE_1 - table container */
+
+			description "TABLE_1 description";
+
+			list TABLE_1_LIST {
+				/* TABLE_1 - object container */
```

Comment:
Can we use some of the real YANGs itself here for generating the config & show commands and verify them by passing various args in the test cases?

Reply:
The unit tests were designed to tests small pieces of `yang_parser.py` logic. For example, I can take the real YANG model, but I also needed to trim the real YANG model content according to the particular unit test. So, in the end, it does not matter if it is a real YANG's or not real.

Reply:
Also, if we will use a real YANG model for the UT, and if this real YANG will be modified in the future, we should also modify the UT.